Wikipedia/UI-V5/WMFArticleContainerViewController.m

```diff
@@ -8,17 +8,23 @@
 
 
 // Controller
-#import "WMFArticleFetcher.h"
 #import "WMFArticleViewController.h"
 #import "WebViewController.h"
 #import "UIViewController+WMFStoryboardUtilities.h"
 #import "WMFSaveButtonController.h"
 #import "WMFPreviewController.h"
 #import "WMFArticleContainerViewController_Transitioning.h"
+#import "WMFArticleHeaderImageGalleryViewController.h"
```

Comment:
Its starting to feel a little crowded in here. Can we do some organization? if not some PONSOs, then maybe some categories. Just a lot of imports now.